### PR TITLE
I created a Typography Page that includes new helpful content

### PR DESCRIPTION
**Pull request by jennyhaines**
## Description
Typography page needed updating.

Verification Conditions:

- plan for PF5 typography styles (but does not need to be included at this time) 
- earlier typography styles are also shown
- General clean up and improvements to how we are showing our typography guidelines
- Address the concern noted here: https://github.com/patternfly/patternfly/issues/351

## Changes

I updated the current PatternFly typographic style, adding in:
2. Table formatting and an image to visuallybreak this page up and make it more digestible
3. New link to MDN's @font-face
4. Descriptions of which styles are _not_ shown on this page, but shown on the PatternFly Typography test page
5. A Simplified set of text style examples
6. The actual details around the text styles (point size, weight, line height, usage)
6. Details around the color usage for text

_**I propose that we include a new tab for the newest Typographic Style once we want that live, and that we mirror the same style as this updated page. (It should be viewed on default, keeping the content of this "old" style in a closed-by-default tab.**_

### CODE REVIEW

**Comment by jennyhaines** [comment]
@LHinson  Here is my pull request containing the typography page I created for [story 2185](https://patternfly.atlassian.net/browse/PTNFLY-2185)

File: pattern-library/navigation/vertical-navigation/design/design.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  - Once the utility items do not fit in the masthead (on small screens), they will responsively move to rest underneath the main navigation items in the vertical navigation.
+  - In any instance that there is an icon present in desktop state with no text label (for example, the help icon), a text label will appear next to that icon in this responsive state.
+
+
+### Secondary Navigation | Responsive State
+![navigation-vertical-secondary-responsive-callout](img/navigation-vertical-secondary-responsive-callout.png)
+1. **Back Link**: Clicking the back link restores the primary-level menu.
+2. **Back Link** (for Utility Item): The back link for a utility item is formatted differently from the back links for main navigational items.
+
+### Notification Drawer | Responsive State
```
Comment (LHinson) [inline pattern-library/navigation/vertical-navigation/design/design.md]: Can we just comment out this part for now?

File: pattern-library/navigation/vertical-navigation/design/design.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+This is how the notification drawer looks on small screens. Once a user selects "Notifications" within a small screen's vertical navigation menu, the notification drawer appears in full-width, with a Back Link above it. [See Here](http://www.patternfly.org/pattern-library/communication/notification-drawer/#/api) for specifics about the Notification Drawer.
+
+![navigation-vertical-notifications-responsive-callout](img/navigation-vertical-notifications-responsive-callout.png)
```
Comment (LHinson) [inline pattern-library/navigation/vertical-navigation/design/design.md]: Comment out for now. Added a story to address the responsive state of the notification drawer and want to come back to this.

File: pattern-library/navigation/vertical-navigation/design/design.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+1. **Back Link**: Clicking the back link restores the primary-level menu.
+2. **Notification Drawer**: The notification drawer takes the entire screen's width when on small screens.
```
Comment (LHinson) [inline pattern-library/navigation/vertical-navigation/design/design.md]: Comment out

**Comment by LHinson** [comment]
@jennyhaines This PR was opened awhile ago and has a bunch of commits (since you were previously poorly setup to contribute) so hard to parse. Can we close this one and open a new PR with your typography updates?

**Comment by LHinson** [comment]
I had a chance to catch up with @jennyhaines in person and she said we were OK to close this one. Thanks!
